src/Client/ConnectionPool.h

```diff
@@ -76,7 +76,7 @@ class ConnectionPool : public IConnectionPool, private PoolBase<Connection>
             Priority priority_ = Priority{1})
        : IConnectionPool(host_, port_),
```

Comment:
Nitpick: Not ideal that there are two unrelated "ConnectionPool" classes. I would probably be pretty confused or annoyed if I encountered it in a codebase. Would it make sense to rename the new ConnectionPool to HTTPConnectionPool or HTTPSessionPool? (Or rename the old one to ClientConnectionPool, but there are 187 occurrences of ConnectionPool/IConnectionPool/ConnectionPoolWithFailover/etc, and people are probably used to that name.)

Reply:
Fixed